qutebrowser/mainwindow/prompt.py

```diff
@@ -467,6 +467,27 @@ def prompt_yank(self, sel=False):
         utils.set_clipboard(question.url, sel)
         message.info("Yanked to {}: {}".format(target, question.url))
 
+    @cmdutils.register(
+        instance='prompt-container', scope='window',
+        modes=[usertypes.KeyMode.prompt])
+    def prompt_external_picker(self):
+        """Choose a location using a configured external picker."""
```

Comment:
Should we abort early if the prompt isn't a `FilenamePrompt`? While the code would in theory work when e.g. being asked for the name of a quickmark, that really doesn't make any sense.

Reply:
Done in 65c3007fc07f708e716c9cb14fefd10bf5f240d7